### DISCUSSION TITLE
Explore tab doesn't have the events

### DISCUSSION THREAD

**Discussion by biel-correa** [community/community#16443] [Mobile] [answered]
It would be great to view all the events in the app since we have a big chance of watching them on our phone.

**Comment by amrehman** [comment, marked as answer]
Hi there @biel-correa , thanks a lot for your feedback.
Can I confirm with you that you are asking for support for GitHub Events in the Explore Tab? (aka https://github.com/events) ? 

Thanks!

**Reply by biel-correa** [reply]
Yes that's exactly what I want

**Reply by amrehman** [reply]
Awesome! I have created an issue for our mobile team. We will jump on this asap! 
Thanks a lot for your feedback!

**Reply by biel-correa** [reply]
Thanks
